site/content/docs/browser-checks/_index.md

```diff
@@ -108,65 +110,108 @@ test('Login to Checkly', async ({ page }) => { // 2
+- [Runtimes](/docs/runtimes/), including the packages in your current runtime
+
+You can use the following keyboard shortcuts to perform routine actions within the Browser check editor.
 
 | Command               | Keybinding                               |
 |-----------------------|------------------------------------------|
 | Save check            | **Mac**: `CMD`+`S` / **Windows**: `CTRL`+`S` |
 | Start/pause check run | **Mac**: `CMD`+`ENTER` / **Windows**: `CTRL`+`ENTER`  |
 | Toggle sidebar        | **Mac**: `CMD`+`B` / **Windows**: `CTRL`+`B` |
 
+### Scheduling & locations
+
+You can configure your checks to run from our [public](/docs/monitoring/global-locations/) locations, or use a Checkly Agent to host your own [private](/docs/private-locations/) locations. If you don't select more than one location and you've disabled retrying checks from the same location, we will pick a random location when retrying checks.
+
+Checkly runs your Browser checks based on an interval you set. The shortest interval you can run is every minute and the longest is every 24 hours.
+
+### Retries & alerting
+
+Select your preferred [retry strategy](/docs/alerting-and-retries/retries/) for failed checks.
+
+Choose which [alert channels](/docs/alerting-and-retries/alert-channels/) to get notified through when your check runs into issues. If we don't have your preferred alert method, use [webhooks](/docs/alerting-and-retries/webhooks/) to configure your alert flow.
+
+### Testing
+
+You can run your check as an [E2E test](/docs/testing) locally or from your CI/CD pipeline to validate your freshly deployed application. Use the Checkly CLI, or configure integrations with Vercel and GitHub.
+
+### SSL certificate domain
+
+You can use Checkly to [monitor SSL certificates](/docs/alerting-and-retries/ssl-expiration/) and get notified before they expire. Since Browser checks can connect to multiple domains, you need to specify which one we should monitor. 
+
+When you change this setting or update your SSL cert, allow up to an hour for this to update on Checkly's end.
+
 ## How do I make assertions?
 
 Navigating around your app or site can already give you a lot of confidence your critical business processes are working correctly.
-However, many times you want to assert specific values on a page.
+However, many times you want to assert specific values on a page. For example:
 
 - After login, you want the user name to be displayed.
 - On a dashboard, you want certain panels to be visible and filled with data.
 - Submitting a form should return a specific value.
 
 To do this, you can:
 
-1. Use the popular [Jest expect](https://jestjs.io/docs/expect) library (Recommended). If you use Playwright Test it is directly available.
+1. Use the popular [Jest expect](https://jestjs.io/docs/expect) library (recommended). If you use Playwright Test, it is directly available.
```

Comment:
@guolau "Use Playwright's built-in expect library" linking to https://playwright.dev/docs/test-assertions, drop the rest (including assert below)